src/editors/containers/TextEditor/hooks.test.jsx

```diff
@@ -1,187 +1,218 @@
-        setEditorRef: jest.fn(),
-        blockValue: blockvalue,
-        openModal: jest.fn(),
-        initializeEditor: jest.fn(),
-        lmsEndpointUrl,
+        blockValue: null,
+        // lmsEndpointUrl: 'sOmEuRl.cOm',
       };
-      let output;
-      test('It creates an onInit which calls initializeEditor and setEditorRef', () => {
+      const evt = 'fakeEvent';
+      const editor = 'myEditor';
+      const addImageUploadBehavior = args => ({ addImageUploadBehvaior: args });
+      beforeEach(() => {
+        props.setEditorRef = jest.fn();
+        props.openModal = jest.fn();
+        props.initializeEditor = jest.fn();
+        jest.spyOn(module, moduleKeys.addImageUploadBehavior)
+          .mockImplementationOnce(addImageUploadBehavior);
         output = module.editorConfig(props);
-        output.onInit();
+      });
+      test('It creates an onInit which calls initializeEditor and setEditorRef', () => {
+        output.onInit(evt, editor);
+        expect(props.setEditorRef).toHaveBeenCalledWith(editor);
         expect(props.initializeEditor).toHaveBeenCalled();
-        expect(props.setEditorRef).toHaveBeenCalled();
       });
       test('It sets the blockvalue to be empty string by default', () => {
-        output = module.editorConfig(props);
         expect(output.initialValue).toBe('');
       });
       test('It sets the blockvalue to be the blockvalue if nonempty', () => {
         const htmltext = 'SomE hTML content';
-        const newprops = {
-          setEditorRef: jest.fn(),
-          blockValue: {
-            data: {
-              data: htmltext,
-            },
-          },
-          openModal: jest.fn(),
-          initializeEditor: jest.fn(),
-        };
-        output = module.editorConfig(newprops);
+        const blockValue = { data: { data: htmltext } };
+        output = module.editorConfig({ ...props, blockValue });
         expect(output.initialValue).toBe(htmltext);
       });
       test('It configures plugins and toolbars correctly', () => {
-        output = module.editorConfig(props);
-        Object.values(module.pluginConfig.plugins).forEach(
-          value => expect(output.init.plugins.includes(value)).toBe(true),
-        );
-        Object.values(module.pluginConfig.toolbar).forEach(
-          value => expect(output.init.toolbar.includes(value)).toBe(true),
-        );
-        Object.values(module.pluginConfig.imageToolbar).forEach(
-          value => expect(output.init.imagetools_toolbar.includes(value)).toBe(true),
+        expect(output.init.plugins).toEqual(pluginConfig.plugins);
+        expect(output.init.imagetools_toolbar).toEqual(pluginConfig.imageToolbar);
+        expect(output.init.toolbar).toEqual(pluginConfig.toolbar);
+        Object.keys(pluginConfig.config).forEach(key => {
+          expect(output.init[key]).toEqual(pluginConfig.config[key]);
+        });
+        //expect(output.init.imagetools_cors_hosts).toMatchObject([props.lmsEndpointUrl]);
```

Comment:
probably should put this test back in, should be a quick fix.

Reply:
I fully pulled out the cors hosts and it still worked, which is why I commented out the test.  will add back in if we decide we need it for the image tools?